arrows/klv/update_klv.cxx

```diff
@@ -176,7 +410,26 @@ vital::config_block_sptr
 update_klv
 ::get_configuration() const
 {
-  return algorithm::get_configuration();
+  auto config = algorithm::get_configuration();
+
+  config->set_value(
+    "st1108_frequency", d->st1108_frequency,
+    "How often (in frames) to encode a ST1108 packet." );
+  config->set_value(
+    "st1108_inter", d->st1108_inter,
+    "How to deal with a group of multiple frames when st1108_frequency > 1. "
+
+    "'sample' will create a packet with the metric values of the first frame "
+    "of the group and associate it with the first frame only, leaving the rest "
+    "of the frames in the group with no associated values. "
+
+    "'sample_smear' will create a packet with the metric values of the first "
```

Comment:
I don't see this addressed anywhere (no check or conditional) or is this the effective default so no checks are required?